spec/components/previews/button_component_preview.rb

```diff
@@ -1,5 +1,5 @@
 class ButtonComponentPreview < ViewComponent::Preview
   def test
-    render(ButtonComponent.new(label: "Awooo!", title: "gaaa!", href: "#", classes: ""))
+    render(ButtonComponent.new(label: "Awooo!", title: "gaaa!", href: "#", scheme: :secondary))
   end
```

Comment:
Maybe we could have two preview methods: `secondary` and `primary`? Or we could expose `scheme` as an argument in the params so Lookbook lets us override it?

Reply:
That sounds nice in theory. In practice, our Lookbook seems to not behave the way the documents state, and half of the previews we have don't even load, so I don't have a lot of motivation to fiddle with this.

I did try to add one method per `scheme` option, but Lookbook only shows me the results of the last method that I define. 🤷🏼‍♀️

Reply:
I got it working, somehow. I think Lookbook reload is doing something funky? And also dynamically declaring methods doesn't seem to work. I hate everything.